copter/source/docs/tuning-process-instructions.rst

```diff
@@ -158,96 +286,95 @@ If the aircraft appears stable enough to attempt autotune follow the instruction
 There a number of problems that can prevent Autotune from providing a good tune. Some of the reason autotune can fail are:
 
 - High levels of gyro noise.
-- Incorrect value of MOT_THST_EXPO.
+- Incorrect value of :ref:`MOT_THST_EXPO <MOT_THST_EXPO>`.
 - Flexible frame or payload mount.
 - Overly flexible vibration isolation mount.
 - Non-linear ESC response.
-- Very low setting for MOT_SPIN_MIN.
+- Very low setting for :ref:`MOT_SPIN_MIN <MOT_SPIN_MIN>`.
 - Overloaded propellers or motors.
 
 If Autotune has failed you will need to do a manual tune.
 
 Some signs that Autotune has been successful are:
 
-- An increase in the values of ATC_ANG_PIT_P and ATC_ANG_RLL_P.
-- ATC_RAT_PIT_D and ATC_RAT_RLL_D are larger than AUTOTUNE_MIN_D.
+- An increase in the values of :ref:`ATC_ANG_PIT_P <ATC_ANG_PIT_P>` and :ref:`ATC_ANG_RLL_P <ATC_ANG_RLL_P>`.
+- :ref:`ATC_RAT_PIT_D <ATC_RAT_PIT_D__AC_AttitudeControl_Multi>` and :ref:`ATC_RAT_RLL_D <ATC_RAT_RLL_D__AC_AttitudeControl_Multi>` are larger than :ref:`AUTOTUNE_MIN_D <AUTOTUNE_MIN_D>`.
 
 Autotune will attempt to tune each axis as tight as the aircraft can tolerate. In some aircraft this can be unnecessarily responsive. A guide for most aircraft:
 
-- ATC_ANG_PIT_P should be reduced to 6 to 10
-- ATC_ANG_RLL_P should be reduced to 6 to 10
-- ATC_ANG_YAW_P should be reduced to 6 to 10
-- ATC_RAT_YAW_P should be reduced to 0.5 to 1
-- ATC_RAT_YAW_I = ATC_RAT_YAW_P x 0.1
+- :ref:`ATC_ANG_PIT_P <ATC_ANG_PIT_P>` should be reduced from 6 to 10
+- :ref:`ATC_ANG_RLL_P <ATC_ANG_RLL_P>` should be reduced from 6 to 10
+- :ref:`ATC_ANG_YAW_P <ATC_ANG_YAW_P>` should be reduced from 6 to 10
+- :ref:`ATC_RAT_YAW_P <ATC_RAT_YAW_P__AC_AttitudeControl_Multi>` should be reduced from 0.5 to 1
+- :ref:`ATC_RAT_YAW_I <ATC_RAT_YAW_I__AC_AttitudeControl_Multi>` = :ref:`ATC_RAT_YAW_P <ATC_RAT_YAW_P__AC_AttitudeControl_Multi>` x 0.1
 
 These values should only be changed if Autotune produces higher values. Small aerobatic aircraft may prefer to keep these values as high as possible.
 
 Setting the input shaping parameters
 ------------------------------------
 
-Arducopter has a set of parameters that define the way the aircraft feels to fly. This lets allows the aircraft to be set up with a very aggressive tune but still feel like a very docile and friendly aircraft to fly. 
+Arducopter has a set of parameters that define the way the aircraft feels to fly. This lets allows the aircraft to be set up with a very aggressive tune but still feel like a very docile and friendly aircraft to fly.
```

Comment:
This allows the aircraft to be set up with a very aggressive tune but still feel like a very docile and friendly aircraft to fly.